contentcuration/contentcuration/tests/test_garbage_collect.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+from cStringIO import StringIO
+from datetime import datetime, timedelta
+
+import requests
+from django.conf import settings
+from django.core.files.base import ContentFile
+
+from contentcuration.models import ContentNode, File
+from contentcuration.utils.garbage_collect import clean_up_contentnodes
+
+from base import StudioTestCase
+
+
+THREE_MONTHS_AGO = datetime.now() - timedelta(days=93)
```

Comment:
Do we want this value to be something under settings.py? (`days=settings.BUFFER_DELETE`)